lib/component.js

```diff
@@ -68,11 +69,29 @@ function build ( string, opts, readFile ) {
 						promises.push( readFile( src ).then( str => partials[id] = str ) );
 					}
 				} else if ( !type || type === 'text/javascript' || type === 'application/javascript' ) {
+					const rel = getAttr( 'rel', item);
+
+					if ( rel === 'css' ) {
+						if ( cssFn ) return Promise.reject( `Only one css script may be specified per component.` );
+						else if ( style.length ) return Promise.reject( `<style> elements and css script cannot be mixed.` );
+
+						cssFn = true;
+					}
+
 					if ( !src ) {
-						script.unshift( item.f );
+						if ( rel === 'css' ) {
+							style.unshift( makeCssFn( item.f.join('') ) );
+						} else {
+							script.unshift( item.f );
+						}
```

Comment:
For this one, `<link rel="text/ractive-fn-css" href="...">`. Would make sense since this is how external CSS is loaded, plus `<link>` fits since it's a multi-purpose element for loading external things (favicon, etc.).